### ISSUE TITLE
SIP still uses package source when running under gitified

### ISSUE THREAD

**Issue by mrenvoize** [mkfifo/koha-gitify#11]
Currently, it's rather challenging to test code that changes the SIP services under a gitified environment.

When the koha-sip script is envoked, it works with code in /usr/lib/koha instead of the gitified code.

**Comment by wizzyrea** [comment]
What I did on the gitified dev install is simply change the environment path in the koha-start-sip script.  Yes, it affects all of the instances, but since it was a test box it didn't much matter.

**Comment by mkfifo** [comment]
@wizzyrea yeah that is an easy fix

but for the general case (of wanting to mix gitified and non-gitified) we would most likely need an upstream change for the various cron jobs to obey some path setting (so we could set it per instance)

this general case would also work if we wanted many gitified instances but each with a different path.

**Comment by wizzyrea** [comment]
Yeah, I was just noting it down for anyone else who needed/wanted a workaround :)

**Comment by mkfifo** [comment]
@wizzyrea +1

**Comment by mrenvoize** [comment]
I never got back to this one.. sorry.

I'm using gitified less and less to be honest. I think we're a fringe case, but we come up against allot of these issues pretty often.  It's still a great tool, and it's great to see people using it so much.. but for my dev and qa tasks I'll be sticking to full dev type installs for a while yet.
